### ISSUE TITLE
cron in 10.3 is broken

### ISSUE THREAD

**Issue by Chi-Yu** [owncloud/core#36316]
### Steps to reproduce
1. Upgrade to **10.3**.
2. Get this via email every 15 minutes: `sh: 1: ./occ: not found`
3. Refer to the **[admin documentation](https://doc.owncloud.com/server/10.3/admin_manual/configuration/server/background_jobs_configuration.html)** for **10.3** to verify the cron setup didn't change, just to realize that the admin docs have **not been updated** to reflect the changes.
4. Find some random dude having the same problem on **[central.owncloud.org](https://central.owncloud.org/t/occ-not-working-after-update-to-10-3/22338)** (which I don't know what it is since that site comes without any information were you just landed) .
5. Read the linked update notes which you didn't read because you are using the debian package and expect things to work automatically (which they don't).
6. Actually **update** your cronjob according to the update notes.
7. Get **this** via email every 15 minutes:
    ```
    1 [->--------------------------]
    2 [-->-------------------------]
    3 [--->------------------------]
    4 [---->-----------------------]
    5 [----->----------------------]
    6 [------>---------------------]
    7 [------->--------------------]
    8 [-------->-------------------]
    9 [--------->------------------]
   10 [---------->-----------------]
   11 [----------->----------------]
   12 [------------>---------------]
   13 [------------->--------------]
   14 [-------------->-------------]
   15 [--------------->------------]
   16 [---------------->-----------]
    ```

### Expected behaviour
1. Working fallback code which doesn't die with `not found` errors
2. Up-to-date admin manual
3. Notes during the update procedure which actually tell me about manual steps I need to take
4. Proper cronjob code which is designed to be run by a cron daemon

### Actual behaviour
1. Broken fallback code which floods my mailbox with errors
2. Outdated admin manual
3. Update procedure which happily claims that everything went well and there is nothing left to do
4. Cronjob code renders a progress bar which is designed to be consumed by humans, not a cron daemon.

### Server configuration
**Operating system**:
Debian 9.11

**Web server:**
Apache 2

**Database:**
*Not relevant*

**PHP version:**
7.0.33-0+deb9u5

**ownCloud version:**
10.3

**Updated from an older ownCloud or fresh install:**
updated

**Where did you install ownCloud from:**
Debian package

**Signing status (ownCloud 9.0 and above):**
> No errors have been found.

**The content of config/config.php:**
*Not relevant*

**List of activated apps:**
*Not relevant*

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** yes/no
*Not relevant*

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No

### Client configuration
**Browser:**
*Not relevant*

**Operating system:**
*Not relevant*

#### Web server error log
*Not relevant*

#### ownCloud log (data/owncloud.log)
*Too large to be posted since log rotation is off by default*

#### Browser log
*Not relevant*

**Comment by DeepDiver1975** [comment]
Will be fixed in 10.3.1 - pr was merged today.

**Comment by ho4ho** [comment]
> that the admin docs have not been updated to reflect the changes.

Documentation by https://github.com/owncloud/docs/pull/1234 updated?
